src/bilder/images/edx_notes/files/docker-compose.yaml

```diff
@@ -2,27 +2,47 @@
 services:
   notes_api:
     image: 610119931565.dkr.ecr.us-east-1.amazonaws.com/dockerhub/mitodl/openedx-notes:${OPENEDX_RELEASE}
-    ports:
-    - 8000
+    depends_on:
+    - notes_django_migrate
+    volumes:
+      # Ref: CONFIG_ROOT
+      # https://github.com/openedx/edx-notes-api/blob/master/notesserver/settings/yaml_config.py
+    - /etc/docker/compose/edx_notes_settings.yaml:/app/edx-notes-api/edx_notes_api.yml:ro
     env_file:
     - ./.env
+    ports:
+    - '8000'
     labels:
     - "traefik.enable=true"
     - "traefik.http.routers.notes_api.rule=Host(`${NOTES_API_HOST}`)"
     - "traefik.http.routers.notes_api.tls.certresolver=letsencrypt_resolver"
     - "traefik.http.routers.notes_api.entrypoints=https"
+  notes_django_migrate:
+    image: 610119931565.dkr.ecr.us-east-1.amazonaws.com/dockerhub/mitodl/openedx-notes:${OPENEDX_RELEASE}
+    depends_on:
+    - notes_es_crate_index
```

Comment:
I think it makes sense to reverse this dependency, in case there are ever search index changes that depend on structural updates to the application database.